controllers/topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../models/topics.models");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics().then((response) => {
+    res.status(200).send(response);
```

Comment:
you should send back all data from your controller on a meaningful key eg. `.send({topics: response})`